libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/TabsView.tsx

```diff
@@ -104,44 +109,76 @@ export class TabsView extends React.Component<ITabsViewProps, ITabViewState> {
               tokens={stackTokens}
               className={classNames.mainImageContainer}
             >
-              <Stack className={classNames.halfContainer} tokens={stackTokens}>
-                <Stack.Item id="errorInstances">
-                  <TitleBar
-                    count={this.props.errorInstances.length}
-                    type={TitleBarOptions.Error}
-                  />
-                </Stack.Item>
-                <Stack.Item
-                  className={classNames.imageListContainer}
-                  id="errorImageContainer"
+              {this.props.taskType === DatasetTaskType.ObjectDetection ? (
```

Comment:
did you make sure that the non-object detection cases are still working exactly the same as before?  asking because we had several cases in the past where object detection UI changes broke the multiclass/multilabel scenarios.